migrations/1592258231605_reminders.ts

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import { MigrationBuilder, ColumnDefinitions } from 'node-pg-migrate';
+
+export const shorthands: ColumnDefinitions | undefined = undefined;
+
+const autoUpdatingTimestamp = (pgm: MigrationBuilder) => ({
+    type: "timestamp",
+    notNull: true,
+    default: pgm.func("current_timestamp")
+});
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+    pgm.createTable("reminder", {
+        id: "id",
+        created_at: autoUpdatingTimestamp(pgm),
+        last_updated: autoUpdatingTimestamp(pgm),
+        workspace_id: {type: "varchar(1000)", notNull: true},
+        channel_id: {type: "varchar(1000)", notNull: true},
```

Comment:
If this combination is going to be commonly used in action raptor, we should consider creating another table that stores unique combinations and is referenceable via foreign key. As a bonus, this will create an index for this field which will make it easy to search

Reply:
yeah that totally crossed my mind too. Makes sense